A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -180,16 +180,16 @@ class CDUPerformancePage {
         }
 
         // transition altitude - remains editable during take off
-        let transAltCell = "";
+        let transAltCell = "{cyan}[  ]{end}";
```

Comment:
Are you sure this is accurate? @beheh and @MisterChocker have done quite a bit of work to make this page accurate. Also, only use non-breaking spaces.

Reply:
Yes IRL Pilot shows to me that space is cyan [     ] when remove it

Reply:
Honeywell, not Thales?

Reply:
Yes, we have a Honeywell reference: https://discord.com/channels/738864299392630914/754851312306487326/822059909826871306